src/cartesian/Brush.tsx

```diff
@@ -127,6 +129,7 @@ export class Brush extends PureComponent<Props, State> {
     padding: { top: 1, right: 1, bottom: 1, left: 1 },
     leaveTimeOut: 1000,
     alwaysShowText: false,
+    ariaLabel: 'chart brush',
```

Comment:
isn't this now misleading? The users `ariaLabel` won't have full control on what actually ends up in the label. It will be put into the "ID" of the `ariaLabelBrush`. We should have what is in `ariaLabelBrush` be the default value and allow the user to override the entire thing, rather than just a portion of it.